src/components/__tests__/IssueSearchAdvanced.spec.ts

```diff
@@ -32,4 +32,32 @@ describe('IssueSearchAdvanced', () => {
   it('should render', () => {
     expect(wrapper.exists()).toBe(true);
   });
+
+  it('should show advanced search', async () => {
+    const content = wrapper.find('details');
+    expect(content.exists()).toBe(true);
+  });
+
+  it('should emit on search button click', async () => {
+    const searchButton = wrapper.find('button[type="submit"]');
+    expect(searchButton.exists()).toBeTruthy();
+    await searchButton.trigger('submit');
+    await flushPromises();
+    const setFilterEvent = wrapper.emitted('set:filters');
+    expect(setFilterEvent[0][0]).toEqual({});
+  });
+
+  it('should emit with filters on search button click', async () => {
```

Comment:
If you end up changing the test below, this description should change to reflect the nature of the test